Neos.Eel/Classes/Helper/MathHelper.php

```diff
@@ -398,7 +398,7 @@ public function randomInt($min, $max)
      * The precision defines the number of digits after the decimal point.
      * Negative values are also supported (-1 rounds to full 10ths).
      *
-     * @param float $subject The value to round
+     * @param mixed $subject The value to round
```

Comment:
This makes Psalm happy and is technically correct, as the method handles any type.